src/Modules/LocalRecordsModule/LocalRecordsModule.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <RootNamespace>EvoSC.Modules.Official.LocalRecordsModule</RootNamespace>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AssemblyName>LocalRecordsModule</AssemblyName>
+    <Title>Local Records</Title>
+    <Description>Determine and display top records of a map from locally saved records.</Description>
+    <Version>1.0.0</Version>
+    <Authors>Evo</Authors>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Templates\**\*" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="Localization.resx" />
+  </ItemGroup>
+  <ItemGroup>
+        <ProjectReference Include="../../EvoSC.Modules.SourceGeneration/EvoSC.Modules.SourceGeneration.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+        <ProjectReference Include="../../EvoSC.Modules/EvoSC.Modules.csproj" Private="true" />
+        <ProjectReference Include="..\PlayerRecords\PlayerRecords.csproj" />
```

Comment:
Not to be nitpicky, but we should probably check that all these are using forward slashes for consistency sake.